chingu-voyages/v50-tier2-team-14
# Feature/hero page

# Description
Created Hero.js UI component as a welcome page for the app. On clicking the button, the user will be taken to the pain page of the app.


## Main changes explained:
- Created Hero.js
- Added Rabbit logo with 'Hungry?' text to src/images folder
- Displayed Hero.js in App.js conditionally
…

## How to test:
1. check into current branch
2. do `npm install` and `npm start` to run this PR locally
3. go to `localhost:3000`
4. check that you see the page below:

![Screenshot 2024-08-06 at 12 03 17 PM](https://github.com/user-attachments/assets/a4089f50-468c-472b-875b-8bb36d4a7197)

5.Verify that  you are taken to the homepage (menu page for now) when clicking the button
